tv_apps/tv-ftu/index.html

```diff
@@ -37,25 +38,81 @@
 
   <body>
 
-    <article id="app-usage-slide" class="slide">
-      <section class="slide-section top">
-        <img id="app-usage-slide-logo" src="style/images/firefox_logo.png">
+    <article id="app-usage-page" class="page">
+      <section class="page-section logo-section">
+        <img class="firefox-logo" src="style/images/firefox_logo.png">
+        <span class="txt-large txt-center" data-l10n-id="welcome"></span>
       </section>
-      <section id="app-usage-terms-section" class="slide-section mid" tabindex="-1">
-        <div id="app-usage-terms">
-          <p id="app-usage-terms-title" class="slide-description txt-large txt-center" data-l10n-id="about-firefox-os"></p>
-          <p class="slide-description txt-mid" data-l10n-id="welcome-to-fx-os"></p>
-          <p class="slide-description txt-mid" data-l10n-id="mozilla-cares-privacy"></p>
-          <p class="slide-description txt-mid" data-l10n-id="initial-activation"></p>
-          <p class="slide-description txt-mid" data-l10n-id="app-usage"></p>
-          <p class="slide-description txt-mid" data-l10n-id="tv-is-created-by-several-parties"></p>
-          <p class="slide-description txt-mid txt-center" data-l10n-id="send-data"></p>
+      <section class="page-section content-section" tabindex="-1">
+        <div class="page-content">
+          <p class="page-description" data-l10n-id="fxos-is-free"></p>
+          <p class="page-description" data-l10n-id="app-usage-notice"></p>
+          <p class="page-description" data-l10n-id="only-cover-mozilla"></p>
+          <p class="page-description txt-center" data-l10n-id="send-to-mozilla"></p>
         </div>
       </section>
-      <section class="slide-section bottom">
-        <div id="app-usage-arrow-icon" data-icon="arrow-down"></div>
-        <smart-button id="app-usage-no-button" type="rounded-corner-text" data-l10n-id="no"></smart-button>
-        <smart-button id="app-usage-yes-button" class="primary" type="rounded-corner-text" data-l10n-id="yes"></smart-button>
+      <section class="page-section button-section">
+        <div class="arrow-icon hidden" data-icon="arrow-down"></div>
+        <smart-button id="app-usage-page-no-button" class="page-button" type="rounded-corner-text" data-l10n-id="no"></smart-button>
+        <smart-button id="app-usage-page-yes-button" class="primary page-button page-default-button" type="rounded-corner-text" data-l10n-id="yes"></smart-button>
+      </section>
+    </article>
+
+    <article id="privacy-policy-page" class="page hidden">
+      <section class="page-section content-section" tabindex="-1">
+        <div class="page-content">
```

Comment:
So this is only one FTE page, but it's actually two pages? I think this is somewhat confusing since you are mixing your own paging mechanism together with FTE-wizard.